<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>442</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Trebuchet MS</family>
    <pointsize>8</pointsize>
    <stylestrategy>PreferDefault</stylestrategy>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Simple OBJ Viewer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="3" rowspan="3">
     <widget class="QGraphicsView" name="view">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="interactive">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QGroupBox" name="leftBox">
      <property name="minimumSize">
       <size>
        <width>147</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>147</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="loadButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Load a file...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="quitButton">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="4" rowspan="3">
     <widget class="QGroupBox" name="rightBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="6" column="0">
        <widget class="QGroupBox" name="translationBox">
         <property name="title">
          <string>Translate</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="translateXButton">
            <property name="text">
             <string>Translate X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="translateYButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">alternate-background-color: rgb(255, 255, 0);</string>
            </property>
            <property name="text">
             <string>Translate Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="translateZButton">
            <property name="text">
             <string>Translate Z</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QGroupBox" name="scalingBox">
         <property name="title">
          <string>Scale</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="scalePlusButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Scale +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="scaleMinusButton">
            <property name="text">
             <string>Scale -</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="rotationBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string>Rotate</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="clockwiseCheck">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>clockwise</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rotateXButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Rotate around &amp;x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rotateYButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Rotate around &amp;y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rotateZButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Rotate around &amp;z</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="reloadButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Reset the model to its original state.</string>
         </property>
         <property name="text">
          <string>Reload model</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <kerning>false</kerning>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
     </font>
    </property>
    <property name="title">
     <string>Model</string>
    </property>
    <addaction name="actionReload"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuModel"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose an OBJ file to be opened.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reset the model to its original state.</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
